dali/operators/reader/loader/webdataset_loader.h

```diff
@@ -67,14 +67,17 @@ class VectorRange {
 };
 
 struct ComponentDesc {
-  std::string ext;
+  std::string filename, ext;
```

Comment:
Is that the name without extension? In WDS nomenclature it's called `key`.
```suggestion
  std::string key, ext;
```

Reply:
> is that the name without extension?

It is actually the name with the extension

Reply:
About 
> Is this change reflected somehow in the indexed version?

I don't fully understand, what you have in mind here

Reply:
I see now the index supports it.